pkg/template/registry/rest.go

```diff
@@ -94,6 +94,7 @@ func (s *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		// TODO: We don't report the processing errors to users as there is no
```

Comment:
And remove the comment.  Also, you should never return an aggregate error, you should be returning a 422 via errors.NewInvalid(...)